students/meet_shah/lesson05/mailroom3.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+"""
+simple mailroom program for a non-profit
+"""
+
+import sys
+
+donors = {"William Gates, III": [10000, 2000000, 5000, 550000],
+          "Mark Zuckerberg": [10, 100, 25, 100],
+          "Jeff Bezos": [50,  10, 15, 100],
+        }
+
+def thank_you():
+    while True:
+        new_response = input("Enter a command. Your options are [list, <donor name>, q(quit),]>>")
+        if new_response == "list":
+            print("\nHere's a list of our donors:\n")
+
+            ### Added list comprehension ###
+            [print(donor) for donor, _ in donors.items()]
```

Comment:
Nice! Since you don't need the value. it'd be slightly more efficient to do
```
[print(donor) for donor in donors]
```
Because if you don't specify, it'll just give you the key, and some teacher once told me that ways more efficient than specifically telling it to give the donors.keys().